src/SprykerEco/Shared/Heidelpay/HeidelpayConfig.php

```diff
@@ -105,6 +136,16 @@ class HeidelpayConfig extends AbstractBundleConfig
      */
     public const CAPTURE_TRANSACTION_STATUS_OK = 'ACK';
 
+    /**
+     * Specification:
+     * - The response of transaction failed status.
+     *
+     * @api
```

Comment:
Please, add also `@api` tag for all cases like this. It's a public API.

Reply:
fixed.